lib/migration.js

```diff
@@ -134,7 +136,69 @@ Migration.prototype.sqlFileLoaderTemplate = function() {
     "};",
     "",
     "exports.down = function(db, callback) {",
-    "  var filePath = path.join(__dirname + '/sqls/"+this.name.replace('.js', '')+"-down.sql');",
+    "  var filePath = path.join(__dirname + '/sqls/" + this.name.replace(
+      '.js', '') + "-down.sql');",
+    "  fs.readFile(filePath, {encoding: 'utf-8'}, function(err,data){",
+    "    if (err) return callback(err);",
+    "    console.log('received data: ' + data);",
+    "",
+    "    db.runSql(data, function(err) {",
+    "      if (err) return callback(err);",
+    "      callback();",
+    "    });",
+    "  });",
+    "};",
+    ""
+  ].join("\n");
+};
+
+Migration.prototype.sqlFileLoaderIgnoreOnInitTemplate = function() {
+  return [
+    "'use strict';",
+    "",
+    "var dbm;",
+    "var type;",
+    "var seed;",
+    "var fs = require('fs');",
+    "var path = require('path');",
+    "var ignoreOnInit = false;",
+
+
+    "",
+    "/**",
+    "  * We receive the dbmigrate dependency from dbmigrate initially.",
+    "  * This enables us to not have to rely on NODE_PATH.",
+    "  */",
+    "exports.setup = function(options, seedLink) {",
+    "  dbm = options.dbmigrate;",
+    "  ignoreOnInit = dbm.getInstance().internals.argv['ignore-on-init']",
```

Comment:
Ok thank you. I think I will take time to review this tomorrow. I think the test and documentation are missing right now. But I will give you feedback as soon as I get to it.